### PR TITLE
Release1.10.0

### PR DESCRIPTION
**Pull request by baude**

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Release Notes
 
+## v1.10.0
+* fixed #411
+* removed unused kill switch
+* updated ependencies
```
Comment (Luap99) [inline RELEASE_NOTES.md:6]: typo

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Release Notes
 
+## v1.10.0
+* fixed #411
```
Comment (Luap99) [inline RELEASE_NOTES.md:4]: there is really no point in mentioning it because it was a CI flake only.

**Comment by lsm5** [comment]
> podman system tests failed on RHEL. @containers/packit-build please check.

Have we seen these before? Wonder if it's aarch64+RHEL combo specific given it passed everywhere else.

**Comment by baude** [comment]
fixed up

**Pull request merged**